common/level.go

```diff
@@ -27,26 +24,26 @@ func (t *tile) Width() float32 {
 }
 
 func (t *tile) Texture() *gl.Texture {
-	return t.Image.Texture()
+	return t.Image.id
 }
 
 func (t *tile) Close() {
 	// noop
 }
 
 func (t *tile) View() (float32, float32, float32, float32) {
-	return t.Image.u, t.Image.v, t.Image.u2, t.Image.v2
+	return t.Image.View()
 }
 
 type tile struct {
-	Point
+	engo.Point
 	height int
 	width  int
-	Image  *Region
+	Image  *Texture
 }
 
 type tilesheet struct {
-	Image    *Texture
+	Image    *TextureResource
```

Comment:
@EtienneBruines Both Texture and TextureResource contain a gl.Texture, but Texture has an additional viewport member along with function implementations for Drawable.  This functionality is not needed in this case as the tilesheet itself will not be rendered.  Let me know if I should use texture anyway.  Tilesheet is used by tmx_level.go along with createTileset.